beaker_server/beaker_cfg/server.cfg

```diff
@@ -10,7 +10,7 @@ server.webpath = "/bkr/"
 # The charset=utf8 option is required for proper Unicode support.
 # The pool_recycle setting is required for MySQL, which will (by default)
 # terminate idle client connections after 10 hours.
-sqlalchemy.dburi = "mysql://beaker:beaker@dci-beaker-containers_db_1/beaker?charset=utf8"
+sqlalchemy.dburi = "mysql://beaker:beaker@172.20.0.10/beaker?charset=utf8"
```

Comment:
How do we know this is the address?

Reply:
This address came from the docker-compose file:
https://github.com/redhat-cip/dci-beaker-containers/blob/main/container-compose.yml#L21

I will try to use a variable instead.